src/rastervision/evaluations/segmentation_evaluation.py

```diff
@@ -0,0 +1,91 @@
+import math
+
+from rastervision.core.class_map import ClassMap
+from rastervision.core.evaluation import Evaluation
+from rastervision.core.evaluation_item import EvaluationItem
+from rastervision.core.label_store import LabelStore
+from rastervision.label_stores.segmentation_raster_file import (
+    SegmentationInputRasterFile)
+from rastervision.raster_sources.image_file import ImageFile
+from rastervision.core.box import Box
+
+
+class SegmentationEvaluation(Evaluation):
+    def compute(self, class_map: ClassMap,
+                ground_truth_label_store: LabelStore,
+                _prediction_label_store: LabelStore) -> None:
+
+        # Construct new prediction label store.  This allows chips to
+        # be read out of the prediction raster with the appropriate
+        # transformations applied.
+        if hasattr(_prediction_label_store, 'source'):
+            raster_source = _prediction_label_store.source
+        elif hasattr(_prediction_label_store, 'sink'):
+            raster_source = ImageFile(None, _prediction_label_store.sink)
+        else:
+            raise ValueError('Must have source or sink.')
+        raster_class_map = ground_truth_label_store.raster_class_map
+        prediction_label_store = SegmentationInputRasterFile(
+            source=raster_source,
+            raster_class_map=raster_class_map)
+
+        # Compute the intersection of the extents of the ground truth
+        # labels and predicted labels.
+        gt_extent = ground_truth_label_store.source.get_extent()
+        pr_extent = prediction_label_store.source.get_extent()
+        xmin = max(gt_extent.xmin, pr_extent.xmin)
+        ymin = max(gt_extent.ymin, pr_extent.ymin)
+        xmax = min(gt_extent.xmax, pr_extent.xmax)
+        ymax = min(gt_extent.ymax, pr_extent.ymax)
+        extent = Box(xmin=xmin, ymin=ymin, xmax=xmax, ymax=ymax)
```

Comment:
Consider extracting a function for computing intersections and moving it to the Box class.

Reply:
:+1: Will do

Reply:
Done.